cache-config/t3c-check/t3c-check.go

```diff
@@ -43,18 +43,18 @@ func main() {
 	flagHelp := getopt.BoolLong("help", 'h', "Print usage information and exit")
 	getopt.Parse()
 	if *flagHelp {
-		fmt.Println(usageStr())
+		log.Errorln(usageStr())
```

Comment:
`t3c-check` hasn't initialized the loggers, so these log lines will never be printed. 

Most apps use an interface, but that's probably overkill here. You can just initialize them all to stderr here:

```
log.Init(os.Stderr, os.Stderr, os.Stderr, os.Stderr, os.Stderr)
```

Reply:
added log.Init()